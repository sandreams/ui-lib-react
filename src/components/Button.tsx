import React from 'react';
const fn = () => {
  console.log(1);
};
const Button: React.FC = () => {
  return (
    <div>
      <button onClick={fn}>按钮</button>
    </div>
  );
};
export default Button;
