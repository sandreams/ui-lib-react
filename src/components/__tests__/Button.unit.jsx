import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button';
describe('Button 组件', () => {
  it('测试 snapshot', () => {
    const component = renderer.create(<Button />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
