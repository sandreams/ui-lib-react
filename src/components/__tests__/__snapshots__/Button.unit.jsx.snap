// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 组件 测试 snapshot 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    按钮
  </button>
</div>
`;
